src/pyodide/pyodide-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+import { loadPyodide, setupPackages } from "pyodide:pyodide-bootstrap-helper";
+import { getMetadata } from "pyodide:current-bundle";
+
+export default {
+  async fetch(request, env) {
+    const pyodide = await loadPyodide();
+    const mainModule = await setupPackages(pyodide, getMetadata());
+    return await mainModule.fetch(request);
+  },
+  async test() {
+    try {
+      const pyodide = await loadPyodide();
+      const mainModule = await setupPackages(pyodide, getMetadata());
+      return await mainModule.test();
+    } catch (e) {
+      console.warn(e);
+    }
+  },
```

Comment:
is this something you want to keep? if so curious why

Reply:
Yeah I think this is useful to keep, for the same reason that the ES6 test entrypoint is useful: it is really helpful to be able to test functionality you use in your `fetch` handler without having to restart the server and refresh `localhost:8080` every time.